src/components/Personalization/createComponent.js

```diff
@@ -113,7 +113,7 @@ export default ({
         });
       },
       onClick({ event, clickedElement }) {
-        onClickHandler({ event, clickedElement });
+        onClickHandler({ event, targetElement: clickedElement });
```

Comment:
What is the reason to rename this?  The click handler in `createOnClickHandler` expects this to be named `clickedElement`.

I suggest reverting the change to this file.

Side note, this change does not actually work.  Since you've modified the onClick lifecycle hook to provie `targetElement`.  To make it work you'd need this code:

```js
      onClick({ event, targetElement }) {
        onClickHandler({ event, targetElement });
      }
```

But still, I suggest you leave this file untouched and rollback your lifecycle hook change to keep the property passed to `onclick` named `clickedElement`.

Reply:
I think you're right. I wasn't sure about this one completely because no unit tests or functional tests failed after changing it. I'd still like to keep it because it mirrors the naming convention of what you get the the click handler API.

Reply:
I'll update the onClick to include targetElement.